app/experimenter/templates/experiments/section_analysis.html

```diff
@@ -7,4 +7,25 @@
 {% block section_content %}
   <p><strong>Data Science Owner: {{ experiment.analysis_owner }}</strong></p>
   {{ experiment.analysis|urlize|linebreaks }}
+
+  {{ experiment.SURVEY_REQUIRED_LABEL }}
+  <strong class="{% if experiment.survey %}text-danger{% endif %}">
```

Comment:
Don't forget to wrap your <strong>'s in <p>'s so you get the nice line break spacing